test/integration_tests/long/functions/test_selfdiffusion.py

```diff
@@ -0,0 +1,61 @@
+# coding=utf-8
+# Copyright 2018-2023 EvaDB
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+from test.markers import stable_diffusion_skip_marker
+from test.util import get_evadb_for_testing
+from unittest.mock import patch
+
+from evadb.server.command_handler import execute_query_fetch_all
+
+
+class StableDiffusionTest(unittest.TestCase):
+    def setUp(self) -> None:
+        self.evadb = get_evadb_for_testing()
+        self.evadb.catalog().reset()
+        create_table_query = """CREATE TABLE IF NOT EXISTS ImageGen (
+             prompt TEXT(100));
+                """
+        execute_query_fetch_all(self.evadb, create_table_query)
+
+        test_prompts = ["pink cat riding a rocket to the moon"]
+
+        for prompt in test_prompts:
+            insert_query = f"""INSERT INTO ImageGen (prompt) VALUES ('{prompt}')"""
+            execute_query_fetch_all(self.evadb, insert_query)
+
+    def tearDown(self) -> None:
+        execute_query_fetch_all(self.evadb, "DROP TABLE IF EXISTS ImageGen;")
+
+    @stable_diffusion_skip_marker
```

Comment:
You also need to add dependency into setup.py. Otherwise, this test will be skipped on circle ci.

Reply:
I did not add the dependency to setup.py because I'm not sure if we want to add the API library that I am using as a core dependency for EvaDB.

Reply:
You do not need to add to core dependency. You can create an extra dependency for this library, and add that to the dev https://github.com/georgia-tech-db/evadb/blob/e8a181c5c34a6e58cc4d3901eae7b20f45b48940/setup.py#L170

So the circle ci will install the dependency and run the test.